### DISCUSSION TITLE
Не приходят монеты

### DISCUSSION THREAD

**Discussion by VIPJeKa** [blockscout/blockscout#9877] [Support]
Я отправил с адреса METAMASK 0xf6431b8c27be24647fedc756d3124a62e454c693 на этот адрес BYBIT 0x4538e4e31ec3aa76fffa24d32bdfdf59d2519ea3 1.893995355711487945 милкАЛГО. В сети эфириума, но монеты так и не дошли до сих пор. Вот хеш транзакции 0x9acca3b80e53e448109370fc9762092f885580f84dfbc69e83fb87fce6e492c2. И скриншот прикрепляю.
![Uploading Screenshot_9.png…]()

**Comment by sl1depengwyn** [comment]
This repo is for Blockscout-related questions. Blockscout is an open source blockchain explorer. Thus, your question is out of scope, where the Blockscout team can help, you better contact the chain support directly.
